pcsx2/GS/Renderers/HW/GSRendererNew.cpp

```diff
@@ -165,7 +165,7 @@ void GSRendererNew::EmulateZbuffer()
 		}
 		else if (!m_context->ZBUF.ZMSK)
 		{
-			m_conf.cb_ps.max_depth = max_z;
+			m_conf.cb_ps.TA_MaxDepth_Af.z = max_z;
```

Comment:
```suggestion
			m_conf.cb_ps.TA_MaxDepth_Af.z = max_z * 0x1p-32f;
```
Old was int, new is float